static/app/views/explore/components/schemaHintsDrawer.tsx

```diff
@@ -0,0 +1,116 @@
+import {Fragment, useMemo} from 'react';
+import styled from '@emotion/styled';
+
+import {Tag as Badge} from 'sentry/components/core/badge/tag';
+import MultipleCheckbox from 'sentry/components/forms/controls/multipleCheckbox';
+import {DrawerBody, DrawerHeader} from 'sentry/components/globalDrawer/components';
+import {IconSearch} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {Tag} from 'sentry/types/group';
+import {prettifyTagKey} from 'sentry/utils/discover/fields';
+import {FieldKind} from 'sentry/utils/fields';
+import {MutableSearch} from 'sentry/utils/tokenizeSearch';
+import {useExploreQuery} from 'sentry/views/explore/contexts/pageParamsContext';
+
+type SchemaHintsDrawerProps = {
+  hints: Tag[];
+};
+
+function SchemaHintsDrawer({hints}: SchemaHintsDrawerProps) {
+  const exploreQuery = useExploreQuery();
+
+  const selectedFilterKeys = useMemo(() => {
+    const filterQuery = new MutableSearch(exploreQuery);
+    return filterQuery.getFilterKeys();
+  }, [exploreQuery]);
+
+  const sortedHints = useMemo(() => {
+    return hints.sort((a, b) => {
```

Comment:
FYI this [sorts `hints` in place](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort). I think you actually want [`toSorted`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toSorted)